packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/android/enrollmentintegrationsteps.md

```diff
@@ -0,0 +1,447 @@
+> **Note:** The examples in this guide use dynamic UI components to build authenticator enrollment and challenge views based on remediations sent in server response by IDX SDK.
+
+### 1: Display the initial sign-in UI
+
+Enable the user to start the sign-in flow, such as presenting a Sign In button or a form with fields for a username and password (you won't use those values until step 3 when you [authenticate the user credentials](#_3-authenticate-the-user-credentials)).
+
+### 2: Start the sign-on flow
+
+Initialize the SDK client and request the initial server response after the user starts the sign-in process. In the example app this is done in a [view model class](https://github.com/okta-samples/okta-android-oie-authenticators-quickstart/blob/main/app/src/main/java/com/okta/android/samples/authenticator/ui/login/LoginViewModel.kt)
+
+```kotlin
+@Volatile
+private var client: IdxClient? = null
+
+...
+private fun createClient() {
+    viewModelScope.launch {
+        // initiate the IDX client and start IDX flow
+        when (val clientResult = IdxClient.start(IdxClientConfigurationProvider.get())) {
+            is IdxClientResult.Error -> {
+                // handle errors
+            }
+            is IdxClientResult.Success -> {
+                client = clientResult.result
+                // calls the IDX API resume to receive the first IDX response.
+                when (val resumeResult = clientResult.result.resume()) {
+                    is IdxClientResult.Error -> {
+                        // handle errors
+                    }
+                    is IdxClientResult.Success -> {
+                        // handle the response `resumeResult.result`
+                        handleResponse(resumeResult.result)
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+The `IdxClientConfigurationProvider` class used in the call to `IdxClient.start()` contains the configuration information for your Okta org, and is usually initialized from an `okta.properties` file. The call to `clientResult.result.resume()` requests the first response of the sign-sequence.
+
+### 3: Authenticate the user credentials
+
+The SDK uses an `IdxResponse` object to represent the server response for the current sign-on step. It contains a collection of remediations (`IdxRemediation`) that specify the type of step, any general messages, and a nested collection of forms (`IdxRemediation.Form`) containing fields (`IdxRemediation.Form.Field`) that represent the various elements required for the UI, such as an text field with a label or a QR code. Completing a step usually requires setting the value of one or more fields. The remediation is then sent back to the server with the updated values by calling the `proceed()` function. The server processes the values and then sends the next step and the the cycle continues until the user is authenticated.
```

Comment:
This paragraph is pretty dense. I think we're making assumptions about the knowledge of the developer, and we need to define these things before.

We should call out specifically that the `proceed` method is on `IdxClient`.

Reply:
Agree but IMO all these need to be properly documented on the IDX SDK docs, wherever we decide to put it. I don't think this guide would be the right place to document the SDK